### DISCUSSION TITLE
Is zustand officially compatible with preact?

### DISCUSSION THREAD

**Discussion by meglio** [pmndrs/zustand#2758] [Q&A]
Searching for "preact" in your documentation gives zero results. Does it mean it is incompatible, or is it "use at your own risk", or the compatibility is simply unknown and untested?

**Comment by dai-shi** [comment]
It's unknown.
There are some old discussions like #1697, but things have changed.
Zustand v5 uses `useSyncExternalStore` from `'react'`, so it depends on Preact's compatibility with React's uSES.

**Comment by meglio** [comment]
Okay let's find out. I aksed preact here: https://github.com/preactjs/preact/issues/4537

**Reply by dbritto-dev** [reply]
@meglio  -> https://github.com/preactjs/preact/blob/main/compat/src/index.js#L170
